MaterialGraphProject/Assets/UnityShaderEditor/Editor/Data/LightweightPipeline/LightWeightUnlitSubShader.cs

```diff
@@ -184,12 +184,18 @@ public string GetSubshader(UnlitMasterNode masterNode, GenerationMode mode)
             subShader.Indent();
             subShader.AddShaderChunk("Tags{ \"RenderType\" = \"Opaque\" \"RenderPipeline\" = \"LightweightPipeline\"}", true);
 
+            var materialOptions = LightWeightPBRSubShader.GetMaterialOptionsFromAlphaMode(masterNode.alphaMode);
```

Comment:
Not a fan of this referencing PBR subshader as we spoke about, but we can push this and change it with another refactor to blending.

Reply:
fixed